### PR TITLE
[ADMINAPI-631] Claimset - Implement multiple auth strategies support

### PR DESCRIPTION
**Pull request by CSR2017**
@jagudelo-gap @lchavarriaga 

Please help review and merge the changes.
Thanks

### CODE REVIEW

File: Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json [hunk 1]
```diff
@@ -1577,7 +1698,10 @@
 											"        function(claimSet) { return claimSet.id; }\r",
 											"    ).indexOf(pm.collectionVariables.get(\"CreatedClaimSetId\"));\r",
 											"\r",
+											"    console.log(results);\r",
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json:1701]: @CSR2017 
Remove console logs

File: Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json [hunk 2]
```diff
@@ -1577,7 +1698,10 @@
...
+											"\r",
 											"    const result = results[indexOfClaimSet];\r",
+											"    console.log(result);\r",
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json:1704]: @CSR2017 
Remove console.log

File: Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json [hunk 3]
```diff
@@ -1626,7 +1750,7 @@
 								"method": "GET",
 								"header": [],
 								"url": {
-									"raw": "{{API_URL}}/v2/claimSets?offset={{offset}}&limit={{limit}}",
+									"raw": "{{API_URL}}/v2/claimsets?offset={{offset}}&limit={{limit}}",
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json:1753]: @CSR2017 
We changed the endpoints from "claimset" to "claimSet", but most of the services were changed again to "claimset" in this PR, Is that ok?

File: Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json [hunk 4]
```diff
@@ -1798,7 +1947,7 @@
 								"method": "GET",
 								"header": [],
 								"url": {
-									"raw": "{{API_URL}}/v2/claimSets/{{CreatedClaimSetId}}",
+									"raw": "{{API_URL}}/v2/claimsets/{{CreatedClaimSetId}}",
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json:1950]: @CSR2017
We changed the endpoints from "claimset" to "claimSet", but most of the services were changed again to "claimset" in this PR, Is that ok?

**Review by jagudelo-gap — CHANGES_REQUESTED** [review]

File: Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json (hunk 3 above)
Comment (CSR2017) [reply Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json:1753]: @jagudelo-gap 
No Juan, I got the latest and updated the tests. Will verify and update. Thanks

**Review by jagudelo-gap — APPROVED** [review]

**Review by lchavarriaga — APPROVED** [review]

**Pull request merged**